src/test_utils.rs

```diff
@@ -129,7 +129,7 @@ impl TestContext {
                     SELECT id, foreign_id, param
                       FROM jobs
                      WHERE action=?
-                  ORDER BY desired_timestamp;
+                  ORDER BY desired_timestamp DESC;
```

Comment:
@flub (I think that it was you who implemented this) - I believe that this needs a `DESC` in order to actually return the _last_ sent message. Without DESC, my test fails as a previous message is returned (the one saying "ephemeral timer message set by me")

Reply:
Sounds good.  This ordering was added as an afterthought I think, so you might be the first one to rely on this and it probably was wrong.  So this looks good to me